### DISCUSSION TITLE
How to implement custom path? I'm in a hurry, Thank you for your reply!!!!!

### DISCUSSION THREAD

**Discussion by parlay96** [vercel/next.js#22628] [Help] [answered]
How to implement custom path? For example: {path: '/ home.abc ', component: import('../viewPage/home') } 。 Path is something I can control, so how can next do it?

**Comment by smontlouis** [comment]
Can you be a little more specific ?

Every path can be created in the `pages` folder.
What kind of path do you need ? You can create a page like this `[...slug].js` and handle redirect to the right component if needed, depending of the content of the slug array.

**Reply by parlay96** [reply]
{
        source: '/about',
        destination: '/',
      },   What do you mean

**Comment by parlay96** [comment]
Old project path: example: ABC AAA, now project refactoring to keep the same path!    How to implement custom path, {path: '/ home.abc ', component: import('../viewPage/home') } 。 Path is something I can control

**Comment by michrome** [comment, marked as answer]
Maybe [`rewrites`](https://nextjs.org/docs/api-reference/next.config.js/rewrites) would help?

**Reply by parlay96** [reply]
thank you
